conf/ldap_conf.scr

```diff
@@ -27,7 +27,7 @@
 .ldap_conf
 
 `ag_ini(
-  `IniAgent( "/etc/openldap/ldap.conf" ,
+  `IniAgent( "/etc/ldap.conf" ,
```

Comment:
I'm not familiar with LDAP so I have some questions here:
- Is there any possible regression regarding changing the config file path?
- I mean what if a user has already changed something in the `/etc/openldap/ldap.conf` file? Will be that change lost?
- Or might be there a conflict between the files?